source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -823,21 +823,26 @@ def fill_journeys(self, request_type, api_request, instance):
         # is modified by create_next_kraken_request function.
         request = deepcopy(api_request)
 
-        min_journeys_calls = get_or_default(request, '_min_journeys_calls', 1)
-
         responses = []
         nb_try = 0
         nb_qualified_journeys = 0
+        nb_previously_qualified_journeys = 0
+        last_chance_retry = False
 
+        min_journeys_calls = get_or_default(request, '_min_journeys_calls', 1)
```

Comment:
`request.get('_min_journeys_calls', 1)` doesn't work?

Reply:
Not tested. This line was just moved here, it isn't related to this PR.